apps/src/componentLibrary/common/styles/mixins.scss

```diff
@@ -0,0 +1,91 @@
+@import "color", "font";
+
+// Typography - Label styles
+@mixin label-common {
+  font-family: $gotham-bold;
```

Comment:
Thinking about the upcoming font migration, should we create a separate constant like `$default-font-bold` or something that maps to `$gotcham-bold` that we can update later?

Reply:
I was thinking about making that change in scope of font change pr